projects/aws-containers/hello-eks-anywhere/Makefile

```diff
@@ -12,8 +12,6 @@ HAS_LICENSES=false
 
 HAS_HELM_CHART=true
 HELM_DIRECTORY=charts/hello-eks-anywhere
-HELM_DESTINATION_REPOSITORY=$(REPO)
```

Comment:
Nil: I know this repo doesn't need to specify `HELM_CHART_NAMES` due to how the repo was set up. I just find it easier to follow through if it's consistent with all other projects.